### PR TITLE
enh(ui): increase timeline density

### PR DESCRIPTION
**Pull request by kduret**
## Description

* Increase timeline density
* update style to improve readability of essential information (content)
* Remove click listing on chips (+ avoid grey background on click)
* Display properly carriage return of event content

To be discussed with @bdauria 

before : 
image

after : 
image

Refs: MON-6650

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 19.10.x
- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x (master)

<h2> How this pull request can be tested ? </h2>

Check timeline events in resource status view

### CODE REVIEW

**Comment by bdauria** [comment]
This looks good :) 

Can we align the chip to the time ?

File: www/front_src/src/Resources/Details/tabs/Services/List.tsx [hunk 1]
```diff
@@ -52,6 +52,7 @@ const ServiceList = ({
             <Paper className={classes.serviceCard}>
               <div className={classes.serviceDetails}>
                 <StatusChip
+                  clickable={false}
```
Comment (bdauria) [inline www/front_src/src/Resources/Details/tabs/Services/List.tsx:55]: I can’t really think of a place where a StatusChip would be clickable, so we wight want to set it to false by default in @centreon-ui

**Comment by Thebarda** [comment]
This looks good ! What do you think to move the 'alias' chip to the right side ?

**Comment by kduret** [comment]
> This looks good ! What do you think to move the 'alias' chip to the right side ?

I've tried but it's a bit weird on "events" cause there is already the `Tries` property on the right

File: www/front_src/src/Resources/Details/tabs/Services/List.tsx (hunk 1 above)
Comment (kduret) [reply www/front_src/src/Resources/Details/tabs/Services/List.tsx:55]: I thought it was used by BAM for enabled/disabled icon, but it's not.
So I'll move default value to false

**Review by bdauria — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Pull request merged**